packages/console/src/components/IconButton/index.tsx

```diff
@@ -4,12 +4,13 @@ import React, { HTMLProps } from 'react';
 import * as styles from './index.module.scss';
 
 export type Props = Omit<HTMLProps<HTMLButtonElement>, 'size' | 'type'> & {
+  className?: string;
```

Comment:
Can remove this line, Props has classname already.

Reply:
ok